### DISCUSSION TITLE
支付宝付费后没有升级成功

### DISCUSSION THREAD

**Discussion by DDYY12** [getlantern/lantern-forum-cn#697] [Q&A]
支付宝已经显示付费，但没有升级成功，没收到任何提示或反馈，请帮忙处理一下。
邮箱账号：david4@126.com
支付宝凭证如下：
![1](https://github.com/getlantern/lantern-forum-cn/assets/174849441/5033e231-1329-4e98-b5a7-bd170e689560)

**Comment by MaxMKMax** [comment]
@DDYY12 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!
